app/cookies/index.js

```diff
@@ -26,7 +26,11 @@ function updatePolicy (request, h, analytics) {
   h.state('cookies_policy', cookiesPolicy, config)
 }
 
+function validSession (request) {
+  return request.state.session
```

Comment:
Should probably return `request.state?.session` just in case the `state` hasn't been generated